ceno_zkvm/src/structs.rs

```diff
@@ -359,13 +376,17 @@ impl<E: ExtensionField> ZKVMWitnesses<E> {
             cs.zkvm_v1_css.num_structural_witin as usize,
             records,
         )?;
-        assert!(
-            self.num_instances
-                .insert(OC::name(), vec![witness[0].num_instances()])
-                .is_none()
+        let num_instances = vec![witness[0].num_instances()];
+        let input = ChipInput::new(
```

Comment:
we need to assign empty witness as placeholder to `witnesses` to skip respective fixed mles, see 
https://github.com/scroll-tech/ceno/pull/1126/files#r2522083303